DynatraceAppMon/Context.cs

```diff
@@ -250,6 +254,89 @@ internal bool goToMethod(Lookup lookup)
             }
         }
 
+        private bool HasClassFiles(ProjectItem projectItem)
+        {
+            bool classFileOccurred = false;
+            for (int i = 1; i <= projectItem.FileCount; i++)
+            {
+                string fileExtension = Path.GetExtension(projectItem.FileNames[(short)i]);
+                if (fileExtension != null && (fileExtension.Contains(".cs") || fileExtension.Contains(".vb")))
```

Comment:
fileExtension.Contains(".cs") will catch other extensions too, if it is required to do so it is better to list exactly what we are searching for and restrict to these extensions only